In [1]:
import pandas as pd
import numpy as np

In [2]:
df=pd.read_csv('BuyerRatio.csv')

In [3]:
df.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [4]:
df.describe()

East         West        North       South
count    2.000000     2.000000     2.000000    2.000000
mean   242.500000   832.500000   743.500000  410.000000
std    272.236111   976.514465   866.205807  480.832611
min     50.000000   142.000000   131.000000   70.000000
25%    146.250000   487.250000   437.250000  240.000000
50%    242.500000   832.500000   743.500000  410.000000
75%    338.750000  1177.750000  1049.750000  580.000000
max    435.000000  1523.000000  1356.000000  750.000000

In [6]:
East=df['East'].mean()
West=df['West'].mean()
North=df['North'].mean()
South=df['South'].mean()

In [7]:
East

242.5

In [8]:
West

832.5

In [9]:
North

743.5

In [10]:
South

410.0

In [18]:
Male = [50,142,131,70]

In [19]:
Female=[435,1523,1356,750]

In [20]:
Sales=[Male,Female]

In [21]:
print(Sales)

[[50, 142, 131, 70], [435, 1523, 1356, 750]]


In [24]:
from scipy.stats import chi2_contingency

In [25]:
chi2_contingency(Sales)

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

# P value(0.66) is greter the Significance level(0.05). Hence null hypothesis cannot be rejected